### ISSUE TITLE
Migrating routeless apps to Diego doesn't disable health-check

### ISSUE THREAD

**Issue by dmarmugi** [cloudfoundry-incubator/Diego-Enabler#8]
As described in https://github.com/cloudfoundry/diego-design-notes/blob/master/migrating-to-diego.md#running-route-less-applications-such-as-workers-and-schedulers, a DEA `--no-route` is analogous to a Diego `--no-route --set-health-check none`

`cf-migrate-apps diego` should disable the health check on routeless DEA applications

Thanks,
-David

**Comment by dkoper** [comment]
@socalnic @ematpl Wdyt? Kinda relates to the discussion we had a while back about whether `cf push --no-route` should imply health-check none.

**Comment by SocalNick** [comment]
If the plugin can properly determine whether or not to add `--set-health-check none`, would make it more useful.

**Comment by ematpl** [comment]
My principal concern would be that there could be a stopped app that when started and routable should have a port-based health-check, but when migrated doesn't happen to have a route mapped to it right then (maybe it's the inactive half of a blue-green deploy pair, and the blue-green deploy technique is being parsimonious with route mappings). The health-check inference on `cf push` has a similar issue. An appropriate mitigation for that concern might be for the migration commands to report warnings about all apps with their health-checks set to `none`, so that the migrator can more easily discover the subset of apps for which to verify that this is the correct health-check type.

Also, can we use the clearer `process` instead of `none` for this?

Thanks,
Eric

**Comment by dmarmugi** [comment]
Hi all, thanks for considering this. 

I can offer that as a user, I wouldn't have had any problem with it dropping me into an interactive (yes/no/yes-to-all/no-to-all) mode to quickly set the appropriate health-checks per app/space. It's essentially what I had to end up doing. 

Thanks, 
-David

**Comment by dkoper** [comment]
We explored interactive prompts when previously enhancing this plugin, but gave up because of its technical complexity (the plugin itself runs in a separate process).  We'll need a lot more demand before we'd want to revisit that.

So it seems the safest solution would be to display a warning for each app being migrated to Diego that has no mapped routes that health-check-type is set to `none`.
(Will look at the stories to support `process` for the cf CLI and app manifest separately):
```
WARNING: Assuming process health check (type "none") for app with no mapped routes; use `cf set-health-check` to change this: App my-app to Diego/DEA in space myspace / org myorg as diesk
```

**Comment by dkoper** [comment]
@dmarmugi We have released v1.2.3 of the plugin with the warning as described above.
Thanks for bringing it up!
